internal/ethapi/api.coreth_test.go

```diff
@@ -35,13 +38,25 @@ func (b *testSuggestPriceOptionsBackend) PriceOptionsConfig() PriceOptionConfig
 	return b.cfg
 }
 
+func (b *testSuggestPriceOptionsBackend) ChainConfig() *params.ChainConfig {
+	return b.chainCfg
+}
+
+func (b *testSuggestPriceOptionsBackend) CurrentHeader() *types.Header {
+	return &types.Header{Time: 1}
+}
+
 func TestSuggestPriceOptions(t *testing.T) {
 	testCfg := PriceOptionConfig{
 		SlowFeePercentage: 95,
 		FastFeePercentage: 105,
 		MaxBaseFee:        100 * params.GWei,
 		MaxTip:            20 * params.GWei,
 	}
+	minBaseFee := 1 * params.GWei
+	bigMinBaseFee := big.NewInt(int64(minBaseFee))
```

Comment:
nit we could just have this as one variable, for example

```suggestion
	minBaseFee := big.NewInt(int64(1 * params.GWei))
```

same below

Reply:
I could restructure the test helpers, but seems out of scope for this fix